tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -610,17 +542,6 @@ if (opArgs.kind != N_OpArgs) { ToolIO.out.println("Bug: not N_OpArgs node"); };
      } ;
     }
 
-// predicate used in lookahead to discriminate between the Case Separator and
-// the box operator. Returns true if it is most likely not the separator.
-// This is a weak mechanism.
-  boolean boxDisc() {
```

Comment:
This dead code is mentioned in a (presumably) outdated comment below:
```
// boxDisc (riminate) uses preInEmptyTop
// note that junction is processed separately.
```

Reply:
Deleted